src/main/java/io/galeb/manager/entity/Team.java

```diff
@@ -64,4 +64,8 @@ public Team setProjects(Set<Project> projects) {
         return this;
     }
 
+    @Override
+    public String getEnvName() {
+        return null;
```

Comment:
There's a NullPointerException risk. Using generic constant may be a best approach, not?